### ISSUE TITLE
RC1

### ISSUE THREAD

**Issue by herecydev** [leastprivilege/BasicAuthentication.AspNet5#1]
Is this still being actively worked on? RC1 is now out and would be good to get this updated and release through nuget.

**Comment by leastprivilege** [comment]
feel free to send a PR

**Comment by herecydev** [comment]
Will try over the weekend.

**Comment by tomcrusader** [comment]
Yeah, this won't fly with RC 1 (final). For example BasicAuthenticationOptions does not contain AutomaticAuthentication

**Comment by leastprivilege** [comment]
true.

**Comment by leastprivilege** [comment]
try this

https://github.com/blowdart/idunno.Authentication
